mail_debrand/models/mail_render_mixin.py

```diff
@@ -97,3 +97,10 @@ def _render_template(
             orginal_rendered[key] = self.remove_href_odoo(orginal_rendered[key])
 
         return orginal_rendered
+
+    def _replace_local_links(self, html, base_url=None):
+        message = super()._replace_local_links(html)
+        message = re.sub(
```

Comment:
You could add a small comment clarifying why you replace by a div and not by nothing